app/src/pages/inside/dashboardItemPage/modals/common/widgetControls/componentHealthCheckControls.jsx

```diff
@@ -179,6 +185,13 @@ export class ComponentHealthCheckControls extends Component {
                 )}
               />
             </FieldProvider>
+            <FieldProvider
+              name="contentParameters.widgetOptions.excludeSkipped"
+              format={Boolean}
+              className={cx('checkbox')}
+            >
+              <Checkbox>{formatMessage(messages.excludeSkipped)}</Checkbox>
+            </FieldProvider>
```

Comment:
Please use the existing `CheckboxControl` from `src/pages/inside/dashboardItemPage/modals/common/widgetControls/controls/checkboxControl.jsx`.
It's usage example for another widget:
![image](https://github.com/reportportal/service-ui/assets/31861805/174d25b0-0175-401c-8103-fb019662fc93)

Reply:
Done.